test/acceptance/behave/components/ngsiv2/entities/update_or_append_entity_attributes/update_or_append_entity_attributes_traces_in_log.feature

```diff
@@ -83,18 +83,30 @@ Feature: verify fields in log traces with update or append entity attributes req
     When update or append attributes by ID "room_1" and with "normalized" mode
     Then verify that receive an "No Content" http code
     And verify that an entity is updated in mongo
-    And check in log, label "INFO" and message "msg=logMsg.h[1803]: Starting transaction from"
+    And check in log, label "INFO" and message "Starting transaction from"
       | trace    | value              |
       | time     | ignored            |
+      | corr     | ignored            |
       | trans    | ignored            |
       | srv      | pending            |
       | subsrv   | pending            |
       | from     | pending            |
       | function | lmTransactionStart |
       | comp     | Orion              |
-    And check in log, label "INFO" and message "msg=logMsg.h[1887]: Transaction ended"
+    And check in log, label "DEBUG" and message "--------------------- Serving request POST /v2/entities/room_1/attrs -----------------"
```

Comment:
About these "--------------------- Serving request" ...
That is but a LM_T in the broker, it might disappear.
But, this is 'fixed' by adding a comment in the source code (in `lib/rest/rest.cpp`):

    // WARNING: This log message is crucial for the correct function of the Behave tests - CANNOT BE REMOVED